### ISSUE TITLE
Determine new buckets and/or pages needed for presidential guidance

### ISSUE THREAD

**Issue by dorothyyeager** [fecgov/fec-cms#2769]
**What we're after**
Many of the same rules that apply to House and Senate candidates also apply to presidential candidates. After doing a review of H4CC content in #2698 and consulting with internal stakeholders, we're going to include guidance for presidential candidates in the [current buckets set up for House and Senate candidates](https://docs.google.com/document/d/1JosieiF4Rhpehm9vjFToCbdWimbX-2aIm81M43CVvKw/edit?usp=sharing). In many cases, this will be a simple change from "House and Senate candidates" to a more generic way of saying candidates for federal office. 

However, there are some areas unique to presidential candidates (for example: public funding, inaugural committees). It may also turn out that some existing content is best separated into congressional and presidential versions after we get into making revisions more deeply. We need to figure out where exactly to place new material related to presidential items. 

Previous presidential issues: #2651, #2697 , #2698 

To do:
- [x] Determine the topics of the new pages we need.
- [x] Determine if we need to make a public funding or inaugural bucket, or if pages on those areas should be made within "Taking in receipts" or "Making disbursements."
- [x] Note which existing pages are going to be updated.

**Comment by dorothyyeager** [comment]
I'm going to start taking a look and thinking through this task while the other things I have going on are being reviewed. My initial plan is to make a working document with  the candidate bucket that can serve as a starting point to show where new pages/buckets might fit in. I'm thinking that the content team and anyone else who should be included can review my first crack at it and we can take it from there to see if we need a meeting. Sound good @AmyKort?

**Comment by AmyKort** [comment]
That sounds like a great plan. Thanks @dorothyyeager !

**Comment by dorothyyeager** [comment]
We've decided to go with a buckets approach after review by the content team, Information Division and Press Office. The offices and team are now reviewing our working document for the buckets to decide upon topics to update or make new pages for.

**Comment by dorothyyeager** [comment]
We're giving everyone until Friday to respond to our working document.

**Comment by dorothyyeager** [comment]
We've gotten our feedback and are ready to proceed on to the next step. We've decided on two new buckets, a bunch of new pages and a bunch of pages that need revision. We've got it outlined in this [color-coded document](https://docs.google.com/document/d/1umE-yD2bBsOKVG6u45phi5akIWYgSe20YX23qKMIGJ4/edit?usp=sharing).

We'll start working on revising the existing pages in #2806 in 8.6. Our work for this issue is complete, so closing.
